src/components/ActionControls/DrawingControls.scss

```diff
@@ -2,28 +2,19 @@
 
 .ba {
     .ba-action-controls-draw {
-        display: flex;
-    }
+        $button-spacing: 5px;
 
-    .ba-action-controls-draw .btn-plain {
-        flex-grow: 1;
-        padding-left: 5px;
-        padding-right: 5px;
+        display: flex;
+        margin-left: -#{$button-spacing};
+        margin-right: -#{$button-spacing};
 
-        .fill-color {
-            fill: $sevens;
```

Comment:
Is the button fill inherited in some other way then?

Reply:
Do we need to set a fill here?

Reply:
It looked correct when I was testing it. Will verify before merging.

Reply:
Restored the style rule below.